Behat/Context/DrupalExtendedContext.php

```diff
@@ -335,6 +344,62 @@ public function createNodeAuthoredCurrentUser($type, TableNode $nodesTable) {
     }
   }
 
+
+  /**
+   * Deletes Files after each Scenario.
+   *
+   * @AfterScenario
+   */
+  public function cleanFiles() {
+    foreach ($this->files as $k => $v) {
+      file_delete($v);
+    }
+  }
+
+  /**
+   * Creates file in drupal.
+   *
+   * @param string $filename
+   *   The name of the file to create.
+   * @param string directory
+   *   A string containing the files scheme, usually "public://".
+   *
+   * @throws Exception
+   *   Exception file not found.
+   *
+   * @throws Exception
+   *   Exception tfile could not be copied.
+   *
+   * @Given file with name :filename
+   * @Given file with name :filename in the :directory directory
+   */
+  public function createFileWithName($filename, $directory = NULL) {
+
+    if (empty($directory)) {
+      $directory = file_default_scheme() . '://';
+    }
+
+    $destination = $directory . '/' . $filename;
+
+    $absolutePath = $this->getMinkParameter('files_path');
+    $path = $absolutePath .'/'. $filename;
```

Comment:
` $absolutePath .'/'. $filename;`

should be

` $absolutePath . '/' . $filename;`